Nemo11235/CS160-Final-Project
# waiting-screen and countdown timer implementation

Description: 
Once player enters a room, a waiting screen will appear that states "waiting for opponent" and displays the room ID as well as a copy to clipboard button. Once the second player joins the room, the countdown starts. Once countdown reaches zero, both players will be able to view the game content and begin playing.

Steps to Test:
1. start server
2. start react app
3. go to multiplayer (http://localhost:3000/race)
4. enter username and room ID, then press join
5. user will then see the waiting screen as the second player has yet to join
![image](https://user-images.githubusercontent.com/64812549/166158394-1e1d8974-99cd-4e0a-a56b-386f396d2dc0.png)

6. on a different tab, go to multiplier (http://localhost:3000/race)
7. on the new tab repeat step 4 with the **same** room ID
8. once join is pressed, the countdown will start for both players
9. after countdown, the multiplier game mode is revealed and players may begin playing
